components/feature/addons/src/main/res/layout/mozac_feature_addons_unsupported_item.xml

```diff
@@ -12,14 +12,25 @@
     android:orientation="horizontal"
     android:padding="16dp">
 
-    <androidx.appcompat.widget.AppCompatImageView
-        android:id="@+id/add_on_icon"
-        android:layout_width="48dp"
-        android:layout_height="48dp"
+    <androidx.cardview.widget.CardView
+        android:id="@+id/icon_container"
+        android:layout_width="40dp"
+        android:layout_height="40dp"
         android:importantForAccessibility="no"
-        android:scaleType="fitCenter"
-        app:srcCompat="@drawable/mozac_ic_extensions"
-        app:tint="?android:attr/textColorPrimary" />
+        app:cardCornerRadius="20dp"
+        app:cardElevation="0dp">
```

Comment:
Interesting approach using `CardView` as a circular background. Is there any advantage over using a background drawable (which I assume is cheaper since it doesn't need to be part of the layouting)?

Reply:
Thanks! Great point!

I went with the CardView just because the layout for installed addons used it already and it provided the nice round background that [Bram recommended](https://github.com/mozilla-mobile/fenix/issues/8578#issuecomment-652167232).
..background which is different from [the specs](https://app.abstract.com/share/28d0811b-ffed-4863-89c8-8aaa10acb528?mode=build&selected=3366345710-5CF8E47B-A223-4368-BE1C-4EEBF8ACDFAE&sha=e00d4f002ecef58f001903d9111f95b864daa8ae) but looks better.
Going with that light grey background would give us this 
![image](https://user-images.githubusercontent.com/11428869/89640424-3439b280-d8b8-11ea-948d-230b28cdf025.png)

Reply:
Thinking that we could add some transparency to the grey background... just don't know how much.. I'll have to eyeball it.

Reply:
Would need different transparency levels for dark/white themes to get the same effect.
So.. I propose sticking with the CardView.

Reply:
We are also using a `CardView` on [the add-ons list](https://github.com/mozilla-mobile/android-components/blob/0094209ac99076787f4e2d3693fff4aa60c40472/components/feature/addons/src/main/res/layout/mozac_feature_addons_item.xml#L17), for the background we are using [the background provide by the app ](https://github.com/mozilla-mobile/android-components/blob/0094209ac99076787f4e2d3693fff4aa60c40472/components/feature/addons/src/main/java/mozilla/components/feature/addons/ui/AddonsManagerAdapter.kt#L235), I think it would be the same in this case.

Reply:
Based on Bram's design guidelines and after confirming with Arturo I went with removing the CardView and also the programatic background.
Thank you both!